rdrf/rdrf/templatetags/get_translate_value.py

```diff
@@ -1,12 +1,14 @@
 from django import template
 from rdrf.models.definition.models import Registry
+from rdrf.helpers.registry_features import RegistryFeatures
 
 register = template.Library()
 
 
 @register.filter
 def get_translate_value(registry):
+    no_translation = RegistryFeatures.NO_TRANSLATION
```

Comment:
nit: You could just inline this (i.e. use `RegistryFeatures.NO_TRANSLATION` instead of `no_translation`) rather than having a variable. I don't think much is gained by having this in a variable.